scout/constants/gene_tags.py

```diff
@@ -2,17 +2,17 @@
 # which are specified by GENETIC_MODELS (in variant_tags.py).
 # The following models are used while describing inheritance of genes in gene panels
 # It's a custom-compiled list of values
-GENE_CUSTOM_INHERITANCE_MODELS = (
-    ("AD", "Autosomal Dominant"),
-    ("AR", "Autosomal recessive"),
-    ("XL", "X Linked"),
-    ("XD", "X Linked Dominant"),
-    ("XR", "X Linked Recessive"),
-    ("NA", "not available"),
-    ("AD (imprinting)", "Autosomal Dominant (imprinting)"),
-    ("digenic", "Digenic"),
-    ("AEI", "Allelic expression imbalance"),
-    ("other", "Other"),
+GENE_STANDARD_INHERITANCE_MODELS = (
```

Comment:
It is just a word, so doesn't really matter, but Im a little bit afraid we will be confused again. To me, this is still custom as compared to GENETIC_MODELS. The other one is really free text custom!?

Reply:
What about GENE_PANELS_INHERITANCE_MODELS ?